### PR TITLE
Improving Reviewing and Merging section

### PR DESCRIPTION
**Pull request by jeremyf**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -114,23 +114,43 @@ You should also add yourself to the `CONTRIBUTORS.md` file in the root of the pr
...
+### Reviewing and Merging Changes
+
+We adopted [Github's Pull Request Review](https://help.github.com/articles/about-pull-request-reviews/) for our repositories.
+There are several checks that may occur in our repositories:
+
+1. Travis CI - where our automated tests are running
+2. Hound CI - where we check for style violations
+3. Approval Required - Github enforces at least one person approve a pull request. Also, all reviewers that have chimed in must approve.
+4. CodeClimate - is our code remaining healthy (at least according to static code analysis)
+5. And whatever may show up in future dates
```
Comment (jcoyne) [inline CONTRIBUTING.md]: I'd omit this.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -114,23 +114,43 @@ You should also add yourself to the `CONTRIBUTORS.md` file in the root of the pr
...
+
+If one or more of the checks failed (or are incomplete), the code should not be merged (and the UI will not allow it). If all of the checks have passed, then anyone on the project (including the pull request submitter) may merge the code.
```
Comment (mjgiarlo) [inline CONTRIBUTING.md]: I'd like to amend thusly: `s/one or more of the checks/one or more of the REQUIRED checks/` (caps not necessary)

**Comment by cam156** [comment]
Interesting.  Since I approved it the button is green even though @mjgiarlo and @jcoyne asked for changes.  May want to mention that.

**Comment by jcoyne** [comment]
@cam156 I didn't ask for changes, just left a comment.  Also, this repository does not have reviews enabled.

**Comment by jcoyne** [comment]
@cam156 I just enabled branch protection, and now we see your review.

**Pull request merged**